app/src/app/session/session.service.ts

```diff
@@ -172,6 +176,21 @@ export class SessionService {
     );
   }
 
+  verifyEmail(): void {
+    this.http.get('/a/reverify')
+      .pipe(take(1))
+      .subscribe({
+      next: () => {
+        this.dialog.open<EmailVerificationDialog, boolean>(EmailVerificationDialog, {data: true});
+      },
+      error: (err: HttpErrorResponse) => {
+        if (err.status == 409) {
+          this.dialog.open<EmailVerificationDialog, boolean>(EmailVerificationDialog, {data: false});
+        }
+      }
+    });
```

Comment:
The indentation of lines 183 to 191 is wrong.  `next` and `error` must not be at the same level as `subscribe`. Line 191 is closing line 182, therefore they must be aligned.